.pre-commit-config.yaml

```diff
@@ -909,30 +909,6 @@ repos:
   # <---- Lint CI Requirements ---------------------------------------------------------------------------------------
 
   # ----- Changelog ------------------------------------------------------------------------------------------------->
-      - id: pip-tools-compile
-        alias: compile-ci-changelog-3.6-requirements
-        name: Changelog CI Py3.6 Requirements
-        files: ^requirements/static/ci/(changelog\.in|py3\.6/(changelog|linux)\.txt)$
-        pass_filenames: false
-        args:
-          - -v
-          - --py-version=3.6
-          - --platform=linux
-          - --pip-args=--constraint=requirements/static/ci/py{py_version}/linux.txt
-          - requirements/static/ci/changelog.in
-
-      - id: pip-tools-compile
```

Comment:
Why are you removing 3.7? We still suport 3.7

Reply:
This is generating the changelog, which is something the team does, so we have more leeway on what versions we want to support, no?
The less requirements to maintain, the better.